requirements-dev.in

```diff
@@ -4,4 +4,5 @@ pip-tools>=5.5.0
 pytest>=4.6
 pytest-cov
 pytest-trio>=0.5.0
+trio<0.25
```

Comment:
technically this constraint should probably be in setup.py

but I suppose it gets resolved if we get the upcoming PR worked out

Reply:
well, the package itself isn't *non-functioning* - it works plenty well with trio==0.25, and if you specify a value to `strict_exception_groups` there is no change in functionality at all. I could even modify the tests in this PR to pass on both <0.25 and >=0.25 and not have the requirement at all.

But I don't see much reason to push a release without #188 anyhow, so I think it's pretty moot